### PR TITLE
Add some simple logic test, with c++ and python

### PR DESCRIPTION
**Pull request by ypraw**
- c++
  - FizzBuzz Program, 
    example output,
    ```bash
    1 2 Fizz 4 Buzz Fizz 7 8 Fizz Buzz 11 Fizz 13 14 FizzBuzz .....
    ```
- Python
  - Number Pattern
    example output,
    
image

### CODE REVIEW

**Comment by insomnius** [comment]
Wow nice contribution @ypraw, but i think you should resolve conflict with master branch.

**Comment by ypraw** [comment]
hi @insomnius, , I have resolved the conflict, thanks for your time, and advice 

best regrads
@ypraw

File: c++/string/FizzBuzz.cpp [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        
+        std::cout<<"Test Logic"<<std::endl;
+
+        // Applies function fn to each of the elements in the range [first,last).
+        // in this case. logicTest Function to choose conditions that are in accordance
+        // with task
+        std::for_each(myvec.begin(),myvec.end(),logicTest);
+        std::cout<<"\n";
+        return 0;
+}
```
Comment (insomnius) [inline c++/string/FizzBuzz.cpp:50]: A new line here would be good. Here is why it should be a new line for every EOF https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: README.md [hunk 2]
```diff
@@ -75,6 +87,8 @@ In order to run all example in this repository you need to have
   - dictionary
```
Comment (insomnius) [inline README.md:87]: Ow, actually its my fault not catch this before. But can you make the first letter uppercase? I would appreciate it.

**Review by insomnius — COMMENTED** [review]
Some minor review here. Good luck.

**Comment by insomnius** [comment]
Oh, dont forget to add your name in CONTRIBUTOR file.

**Pull request closed without merging**